pkgs/tools/networking/dnscrypt-proxy/2.x/default.nix

```diff
@@ -1,24 +1,19 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoPackage, fetchzip }:
 
 buildGoPackage rec {
-  name = "dnscrypt-proxy2-${version}";
-  version = "2.0.15";
+  name = "dnscrypt-proxy-${version}";
+  version = "2.0.17";
 
   goPackagePath = "github.com/jedisct1/dnscrypt-proxy";
 
-  src = fetchFromGitHub {
-    owner = "jedisct1";
-    repo = "dnscrypt-proxy";
-    rev = "${version}";
-    sha256 = "0iwvndk1h550zmwhwablb0smv9n2l51hqbmzj354mcgi6frnx819";
+  src = fetchzip {
```

Comment:
How is this different from `fetchFromGitHub` btw?

Reply:
Not much, wanted to reuse `goPackagePath`.